lib/tether/ops_linux.go

```diff
@@ -929,6 +935,9 @@ func (t *BaseOperations) Setup(config Config) error {
 		return err
 	}
 
+	// NOTE: ulimit default values should change when we support ulimit configuration
```

Comment:
I believe it's #5353. This is the first step to address 5353 if we decide to support ulimit.